target/linux/ramips/mt7620/config-4.19

```diff
@@ -3,18 +3,11 @@ CONFIG_ARCH_BINFMT_ELF_STATE=y
 CONFIG_ARCH_CLOCKSOURCE_DATA=y
 CONFIG_ARCH_DISCARD_MEMBLOCK=y
 CONFIG_ARCH_HAS_ELF_RANDOMIZE=y
-# CONFIG_ARCH_HAS_GCOV_PROFILE_ALL is not set
 CONFIG_ARCH_HAS_RESET_CONTROLLER=y
-# CONFIG_ARCH_HAS_SG_CHAIN is not set
-# CONFIG_ARCH_HAS_STRICT_KERNEL_RWX is not set
-# CONFIG_ARCH_HAS_STRICT_MODULE_RWX is not set
+# CONFIG_ARCH_HAS_SYNC_DMA_FOR_DEVICE is not set
```

Comment:
This is the output from `make kernel_oldconfig CONFIG_TARGET=subtarget` any reason you disable those options? It would make sense to document the reasons in the commit description. Maybe splitting each subtarget in the separate commit would make sense as well.

```diff
diff --git a/target/linux/ramips/mt7620/config-4.19 b/target/linux/ramips/mt7620/config-4.19
index 2162043bf9b3..0d3e9153ecf2 100644
--- a/target/linux/ramips/mt7620/config-4.19
+++ b/target/linux/ramips/mt7620/config-4.19
@@ -4,7 +4,7 @@ CONFIG_ARCH_CLOCKSOURCE_DATA=y
 CONFIG_ARCH_DISCARD_MEMBLOCK=y
 CONFIG_ARCH_HAS_ELF_RANDOMIZE=y
 CONFIG_ARCH_HAS_RESET_CONTROLLER=y
-# CONFIG_ARCH_HAS_SYNC_DMA_FOR_DEVICE is not set
+CONFIG_ARCH_HAS_SYNC_DMA_FOR_DEVICE=y
 CONFIG_ARCH_HIBERNATION_POSSIBLE=y
 CONFIG_ARCH_MMAP_RND_BITS_MAX=15
 CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MAX=15
@@ -16,7 +16,6 @@ CONFIG_ARCH_USE_QUEUED_SPINLOCKS=y
 CONFIG_ARCH_WANT_IPC_PARSE_VERSION=y
 CONFIG_AT803X_PHY=y
 CONFIG_BLK_MQ_PCI=y
-CONFIG_CC_HAS_ASM_GOTO=y
 CONFIG_CEVT_R4K=y
 CONFIG_CEVT_SYSTICK_QUIRK=y
 CONFIG_CLKDEV_LOOKUP=y
@@ -46,11 +45,11 @@ CONFIG_CRYPTO_RNG2=y
 CONFIG_CRYPTO_WORKQUEUE=y
 CONFIG_CSRC_R4K=y
 CONFIG_DEBUG_PINCTRL=y
-# CONFIG_DMA_DIRECT_OPS is not set
+CONFIG_DMA_DIRECT_OPS=y
 CONFIG_DMA_NONCOHERENT=y
-# CONFIG_DMA_NONCOHERENT_CACHE_SYNC is not set
-# CONFIG_DMA_NONCOHERENT_MMAP is not set
-# CONFIG_DMA_NONCOHERENT_OPS is not set
+CONFIG_DMA_NONCOHERENT_CACHE_SYNC=y
+CONFIG_DMA_NONCOHERENT_MMAP=y
+CONFIG_DMA_NONCOHERENT_OPS=y
 # CONFIG_DMA_RALINK is not set
 # CONFIG_DTB_MT7620A_EVAL is not set
 # CONFIG_DTB_OMEGA2P is not set
@@ -67,13 +66,14 @@ CONFIG_GENERIC_CPU_AUTOPROBE=y
 CONFIG_GENERIC_IRQ_CHIP=y
 CONFIG_GENERIC_IRQ_EFFECTIVE_AFF_MASK=y
 CONFIG_GENERIC_IRQ_SHOW=y
-# CONFIG_GENERIC_LIB_ASHLDI3 is not set
-# CONFIG_GENERIC_LIB_ASHRDI3 is not set
-# CONFIG_GENERIC_LIB_CMPDI2 is not set
-# CONFIG_GENERIC_LIB_LSHRDI3 is not set
-# CONFIG_GENERIC_LIB_UCMPDI2 is not set
+CONFIG_GENERIC_LIB_ASHLDI3=y
+CONFIG_GENERIC_LIB_ASHRDI3=y
+CONFIG_GENERIC_LIB_CMPDI2=y
+CONFIG_GENERIC_LIB_LSHRDI3=y
+CONFIG_GENERIC_LIB_UCMPDI2=y
 CONFIG_GENERIC_PCI_IOMAP=y
 CONFIG_GENERIC_PHY=y
+CONFIG_GENERIC_PINCONF=y
 CONFIG_GENERIC_SCHED_CLOCK=y
 CONFIG_GENERIC_SMP_IDLE_THREAD=y
 CONFIG_GENERIC_TIME_VSYSCALL=y
@@ -117,7 +117,7 @@ CONFIG_HAVE_NET_DSA=y
 CONFIG_HAVE_OPROFILE=y
 CONFIG_HAVE_PERF_EVENTS=y
 CONFIG_HAVE_REGS_AND_STACK_ACCESS_API=y
-# CONFIG_HAVE_RSEQ is not set
+CONFIG_HAVE_RSEQ=y
 CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
 CONFIG_HAVE_VIRT_CPU_ACCOUNTING_GEN=y
 CONFIG_HW_HAS_PCI=y
@@ -131,13 +131,13 @@ CONFIG_IRQ_INTC=y
 CONFIG_IRQ_MIPS_CPU=y
 CONFIG_IRQ_WORK=y
 CONFIG_LIBFDT=y
-# CONFIG_LOCK_DEBUGGING_SUPPORT is not set
+CONFIG_LOCK_DEBUGGING_SUPPORT=y
 CONFIG_MARVELL_PHY=y
 CONFIG_MDIO_BUS=y
 CONFIG_MDIO_DEVICE=y
-# CONFIG_MEMFD_CREATE is not set
+CONFIG_MEMFD_CREATE=y
 CONFIG_MFD_SYSCON=y
-# CONFIG_MIGRATION is not set
+CONFIG_MIGRATION=y
 CONFIG_MIPS=y
 CONFIG_MIPS_ASID_BITS=8
 CONFIG_MIPS_ASID_SHIFT=0
@@ -176,14 +176,14 @@ CONFIG_NET_MEDIATEK_SOC=y
 CONFIG_NET_VENDOR_MEDIATEK=y
 # CONFIG_NET_VENDOR_MEDIATEK_STAGING is not set
 CONFIG_NO_GENERIC_PCI_IOPORT_MAP=y
-# CONFIG_NVMEM is not set
+CONFIG_NVMEM=y
 CONFIG_OF=y
 CONFIG_OF_ADDRESS=y
 CONFIG_OF_EARLY_FLATTREE=y
 CONFIG_OF_FLATTREE=y
 CONFIG_OF_GPIO=y
 CONFIG_OF_IRQ=y
-# CONFIG_OF_KOBJ is not set
+CONFIG_OF_KOBJ=y
 CONFIG_OF_MDIO=y
 CONFIG_OF_NET=y
 CONFIG_PCI=y
@@ -211,7 +211,7 @@ CONFIG_SOC_MT7620=y
 # CONFIG_SOC_RT3883 is not set
 CONFIG_SPI=y
 CONFIG_SPI_MASTER=y
-# CONFIG_SPI_MEM is not set
+CONFIG_SPI_MEM=y
 # CONFIG_SPI_MT7621 is not set
 CONFIG_SPI_RT2880=y
 CONFIG_SRCU=y
```